pkg/common/config/aws_config.go

```diff
@@ -17,6 +17,8 @@ var (
 
 	// AWSVPCSubnetIDs is comma-separated list of strings to specify the subnets for cluster provision
 	AWSVPCSubnetIDs = "config.aws.vpcSubnetIDs"
+
+	AWSCreateVPCSubnetIDs = "config.aws.createVPCSubnetIDs"
```

Comment:
Instead of a `AWSCreateVPCSubnetIDs` config setting, can we create the VPC if the subnet ids are not supplied?

Reply:
I think the logic to affect the cloud providers should be gated behind a flag. 
OSDe2e could adapt something like the "--mode auto" like rosa.